tests/testthat/test-options/man/roxygen/meta.R

```diff
@@ -1,3 +1,6 @@
 list(
-  markdown = TRUE
+  markdown = TRUE,
+  knitr_chunk_options = list(
+    myoption = 1:10
```

Comment:
Maybe also set `comment = "###"` here just to check that these override the chunk defaults?